source/camera/SlopeWalk.cpp

```diff
@@ -0,0 +1,270 @@
+/*
+ *  Copyright (C) 2011-2015  OpenDungeons Team
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "camera/SlopeWalk.h"
+
+#include <algorithm>
+#include <iostream>
+#include <sstream>
+
+SlopeWalk::SlopeWalk():
+    leftVertexPassed(false),
+    rightVertexPassed(false)
+{
+}
+
+void SlopeWalk::buildSlopes()
+{
+    int ii;
+    myArray.sort();
+    myArray.zoom(1.2532);
```

Comment:
Yes, it would be better. And also adding a comment where you define it to try to explain why you used this magic number would be good (so that we know for which reason it may be useful to change it)